### PR TITLE
Removed the object destructuring syntax

### PR DESCRIPTION
**Pull request by carlcheel**
This change enables support for node v4.4.4 LTS

### CODE REVIEW

File: AngularSnapshotSerializer.js [hunk 1]
```diff
@@ -60,4 +61,7 @@ const test = val =>
   typeof val === 'object' &&
   Object.prototype.hasOwnProperty.call(val, 'componentRef');
 
-module.exports = {print, test};
+module.exports = {
+  print: print,
```
Comment (thymikee) [inline AngularSnapshotSerializer.js]: Shorthand properties are not supported in Node 4?

Comment (carlcheel) [inline AngularSnapshotSerializer.js]: Nope, see http://node.green/#ES2015-syntax-destructuring--assignment-object-destructuring-with-primitives

File: setupJest.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+'use strict'
```
Comment (thymikee) [inline setupJest.js]: I think `'use strict';` is now supposed to be referenced in every file. Also would you mind adding a comma and extra newline?

File: HTMLCommentSerializer.js [hunk 3]
```diff
@@ -19,4 +19,7 @@ const test = value =>
 
 const print = () => '';
 
-module.exports = {print, test};
+module.exports = {
+  print: print,
+  test: test
+};
```
Comment (thymikee) [inline HTMLCommentSerializer.js]: Please add this newline back :)

File: AngularSnapshotSerializer.js [hunk 4]
```diff
@@ -60,4 +61,7 @@ const test = val =>
...
+  test: test
+};
```
Comment (thymikee) [inline AngularSnapshotSerializer.js]: newline

File: HTMLCommentSerializer.js (hunk 3 above)
Comment (carlcheel) [inline HTMLCommentSerializer.js]: Oops, sorry about that. Done :thumbsup:

File: AngularSnapshotSerializer.js (hunk 4 above)
Comment (carlcheel) [inline AngularSnapshotSerializer.js]: Oops, sorry about that. Done :thumbsup:

**Comment by thymikee** [comment]
Thanks! :)

**Comment by carlcheel** [comment]
No worries, awesome package, made it so easy to get up and running with Angular 4 and Jest :smile:

**Comment by thymikee** [comment]
Glad it works for you! 🙂

**Comment by thymikee** [comment]
Released a patch with this change.

**Pull request merged**